cpp/src/binaryop/jit/code/operation.cpp

```diff
@@ -286,6 +286,15 @@ R"***(
 
     using RLogicalOr = LogicalOr;
 
+    struct UsrDefOp {
```

Comment:
I think the abbreviation in the name here is unnecessary. Spell it out: `struct UserDefinedOp`